WebApp/ISIS/autoreduce_webapp/autoreduce_webapp/settings.py.template

```diff
@@ -49,10 +49,21 @@ AUTHENTICATION_BACKENDS = (
 )
 LOGIN_URL = '/'
 
-TEMPLATE_PATH = os.path.join(BASE_DIR, 'templates')
-TEMPLATE_DIRS = (
```

Comment:
Perhaps related to this. On line 13 in this file, what is TEMPLATE_DEBUG used for?